integration-tests/integration/features/case-progress.feature

```diff
@@ -95,3 +96,39 @@ Feature: Case progress
     Then I should be on the "Case summary" page
     And I should not see govuk notification banner
 
+  Scenario: Display an alert when multiple notes are written simultaneously without being saved previously
+    Given I am an authenticated user
+    And I click the "Accept analytics cookies" button
+    Then I should not see the cookie banner
+
+    When I navigate to the "/B14LO/hearing/5b9c8c1d-e552-494e-bc90-d475740c64d8/defendant/8597a10b-d330-43e5-80c3-27ce3b46979f/summary" base route
+    Then I should be on the "Case summary" page
+    And I should see back link "Back to cases" with href "/B14LO/cases/$TODAY"
+    And I should see the caption text "URN: 01WW0298121"
+
+    And I should see the following summary list
+      | Name          | Kara Ayers                                                            |
+      | Gender        | Female                                                                |
+      | Date of birth | 31 October 1980 (41 years old)                                        |
+      | Address       | 22 Waldorf Court Cardiff AD21 5DR                                     |
+
+    And I should see the level 2 heading "Case progress"
+    And I should see 6 previous hearings headers
+
+    Then I should see the following hearings with the hearing type label, hearing details and next appearance badge if applicable
+      | Millionth hearing   | Sunday 14 July 2999, Court 1, morning session, Neptune Mags       |                 |
+      | Million-1th hearing | Saturday 14 July 2998, Court 1, morning session, Mars Mags        | NEXT APPEARANCE |
+      | 12th hearing        | Saturday 14 December 2019, Court 2, morning session, Leicester    |                 |
+      | 8th hearing         | Wednesday 14 August 2019, Court 2, morning session, Leicester     |                 |
+      | 7th hearing         | Sunday 14 July 2019, Court 1, morning session, Leicester          |                 |
+      | 5th hearing         | Tuesday 14 May 2019, Court 2, morning session, North Shields      |                 |
+
+
+    And the note with the id "1288880" on hearing "2aa6f5e0-f842-4939-bc6a-01346abc09e3" is filled with the text "I am a first unsaved note"
+    And the note with the id "123650" on hearing "2aa6f5e0-f842-4939-bc6a-01346abc09e3" is filled with the text "A"
+
+    Then should appears a css hidden modal which displays information
```

Comment:
avoid technical words like css where possible? "Then user should be alerted with a popup"? 
If you are keeping it as is please correct appears -> appear